.github/workflows/nightly.yml

```diff
@@ -94,3 +88,19 @@ jobs:
         tag_name: less-v${{ needs.checkver.outputs.new_version }}
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    # Publish to WinGet
+    - name: Publish less to WinGet
+      uses: vedantmgoyal2009/winget-releaser@latest
+      with:
+        identifier: JohnTaylor.less
+        release-tag: less-v${{ needs.checkver.outputs.new_version }}
+        installers-regex: 'less\.exe$'
+        token: ${{ secrets.WINGET_TOKEN }}
+    - name: Publish lesskey to WinGet
+      uses: vedantmgoyal2009/winget-releaser@latest
+      with:
+        identifier: JohnTaylor.lesskey
+        release-tag: less-v${{ needs.checkver.outputs.new_version }}
+        installers-regex: 'lesskey\.exe$'
+        token: ${{ secrets.WINGET_TOKEN }}
```

Comment:
@vedantmgoyal2009 Is it fine to use Winget Releaser multiple times for 2 different packages? You have noted in https://bittu.eu.org/docs/wr-multipackage that updating multiple packages from one repository is not supported, but I don't see why this would fail.

Reply:
I don't see any issues with this. The documentation needs to be updated since changes have been made since then. 

Also, @russellbanks @sitiom should I move documentation to the repo itself and use GitHub Pages to build a webpage for convenience or leave it as it is (https://bittu.eu.org/docs/wr-intro)?

Reply:
Repo itself :) that's the first place I check when using something and there's not too much to document with WinGet Releaser anyway